aquarius/app/chains.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import elasticsearch
+import json
+import logging
+
+from flask import Blueprint, Response
+from aquarius.log import setup_logging
+from aquarius.myapp import app
+from oceandb_driver_interface import OceanDb
+
+setup_logging()
+chains = Blueprint("chains", __name__)
+logger = logging.getLogger("aquarius")
+es_instance = OceanDb(app.config["CONFIG_FILE"]).plugin
+###########################
+# Get Chain List
+###########################
+
+
+@chains.route("/list", methods=["GET"])
+def get_chains_list():
+    """Get chains list
+    ---
+    tags:
+      - chains
+    responses:
+      200:
+        description: successful operation.
+      404:
+        description: No chains are present
+    """
+    try:
+        chains = es_instance.driver.es.get(
+            index=f"{es_instance.driver.db_index}_plus", id="chains", doc_type="_doc"
+        )["_source"]
+        return Response(json.dumps(chains), 200, content_type="application/json")
+    except elasticsearch.exceptions.NotFoundError:
```

Comment:
I would say if you catch this one and KeyError (for the `_source` key), you have caught almost everything. And for the except Exception part, you can use the logic from consume and compute that wraps and logs the issue, returning 503.